tests/test_runner/test_hooks.py

```diff
@@ -1190,13 +1190,21 @@ def test_mlflow_hook(log_model):
 
 
 def test_wandb_hook():
+    from mmcv.utils import scandir
+
     sys.modules['wandb'] = MagicMock()
     runner = _build_demo_runner()
     hook = WandbLoggerHook()
     loader = DataLoader(torch.ones((5, 2)))
 
     runner.register_hook(hook)
     runner.run([loader, loader], [('train', 1), ('val', 1)])
+
+    wandb_artifact = hook.wandb.Artifact(name='artifacts', type='model')
+    for filename in scandir(runner.work_dir, hook.out_suffix, True):
+        local_filepath = osp.join(runner.work_dir, filename)
+        wandb_artifact.add_file(local_filepath)
```

Comment:
Can we set `log_artifact` as `True` in `WandbLoggerHook()` and those lines can be removed?

Reply:
i can set `log_artifact` as `True` in `WandbLoggerHook()` but those lines cannot be removed for the tests to pass.

Reply:
Yet, we can directly use `assert_called()` rather than `assert_called_with`